sdks/java/core/src/main/java/org/apache/beam/sdk/io/GenericRecordWriteConverter.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.sdk.extensions.sql.meta.provider.avro;
+package org.apache.beam.sdk.io;
```

Comment:
This file is moved to the core module, I believe you want to move it to the io module?

Reply:
The reason why I moved it here was to be in the same directory as AvroIO and AvroSchemaCapableIOProvider, which happened to be in core. Do you think I should move it to the io module regardless?

Reply:
I see. I wasn't aware that `AvroIO` is in core. Then moving it here make sense.